frontend/src/components/common/Button/Button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement, MouseEvent, PropsWithChildren } from 'react';
+import styled from 'styled-components';
+
+// @TODO use the barrel index.ts file when bug in https://github.com/typescript-eslint/typescript-eslint/issues/1746 is resolved
+import { orange, grey } from 'styles/colors';
+import { fontSize } from 'styles/typography';
+import { spacing } from 'styles/dimensions';
+
+interface Props {
+  readonly isSecondary?: boolean;
```

Comment:
Out of curiosity, why is this `readonly`?

Reply:
Ah, this was not intentional actually! Will change it.

Even though the value is indeed `readonly`, it would have been more useful in other cases but this one should be fine without!